code/game/objects/items/plushes.dm

```diff
@@ -372,14 +372,14 @@
 	squeak_override = list('sound/weapons/bite.ogg'=1)
 
 /obj/item/toy/plush/bubbleplush
-	name = "bubblegum plushie"
+	name = "\improper Bubblegum plushie"
 	desc = "The friendly red demon that gives good miners gifts."
 	icon_state = "bubbleplush"
-	attack_verb = list("rends")
+	attack_verb = list("rended")
```

Comment:
"rended" or "rent"? Both are valid, what do people like more?

Reply:
definitely rent

Reply:
Yeah, that was my preference too